src/UI/SlotArea.cpp

```diff
@@ -780,6 +781,23 @@ void cSlotAreaCrafting::HandleCraftItem(const cItem & a_Result, cPlayer & a_Play
 
 
 
+void cSlotAreaCrafting::ClearCraftingGrid(cPlayer & a_Player)
+{
+	for (int pos = 1; pos <= m_GridSize * m_GridSize; pos++)
+	{
+		auto Item = GetSlot(pos, a_Player);
+		if (Item->m_ItemCount > 0)
+		{
+			a_Player.GetInventory().AddItem(*Item);
+			cItem ItemEmpty = Item->CopyOne();
+			ItemEmpty.Empty();
+			SetSlot(pos, a_Player, ItemEmpty);
```

Comment:
Is this just clearing the item?
```suggestion
			SetSlot(pos, a_Player, cItem());
```